### PR TITLE
move location polling to App component

### PR DESCRIPTION
**Pull request by steinn**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -10,20 +11,50 @@ console.ignoredYellowBox = ['Warning: View.propTypes'];
...
 
+  componentDidMount() {
+    this.updateLocation();
+    setInterval(() => {
+      this.updateLocation();
+    }, 10000);
+  }
+
+  updateLocation = async () => {
+    console.log('UpdateLocation');
```
Comment (arnarthor) [inline App.js]: Remove

File: App.js [hunk 2]
```diff
@@ -10,20 +11,50 @@ console.ignoredYellowBox = ['Warning: View.propTypes'];
...
+        errorMessage: 'Permission to access location was denied',
+      });
+    }
+
+    let location = await Location.getCurrentPositionAsync({});
+    this.setState({location: location});
+  };
+
   render() {
+    console.log('Render App');
```
Comment (arnarthor) [inline App.js]: Remove

File: CurrentSpeed.js [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
 
-  getLocationAsync = async () => {
-    let {status} = await Permissions.askAsync(Permissions.LOCATION);
-    let location = await Location.getCurrentPositionAsync({});
-    this.setState({location});
-  };
-
   render() {
-    const {location} = this.state;
+    console.log('Render CurrentSpeed');
```
Comment (arnarthor) [inline CurrentSpeed.js]: Remove 😄

File: CurrentSpeed.js [hunk 4]
```diff
@@ -1,40 +1,41 @@
...
         <Text style={styles.text}>
-          {location && location.coords.speed > 0
-            ? Math.ceil(
-                convertMetersPerSecondToKmPerHour(location.coords.speed)
-              )
+          {coords && coords.speed > 0
+            ? Math.ceil(convertMetersPerSecondToKmPerHour(coords.speed))
             : 0}
-          {' '}km/h
+          {' '}km/h{' '}
```
Comment (arnarthor) [inline CurrentSpeed.js]: Why do we need whitespace behind?

Comment (steinn) [inline CurrentSpeed.js]: I don't have a clue, I must have accidentally added this.

**Pull request merged**